### ISSUE TITLE
Theurgy Quest Restructuring

### ISSUE THREAD

**Issue by RedTechGamer236** [TeamAOF/Craftoria#195]
### Describe your suggestion

I suggest restructuring the beginning quests for the Theurgy quest chapter to indicate that you need the calcination oven, sal ammoniac stuff, liquefaction cauldron, and mercury distiller to make everything needed for the incubator.

### Why would you like this added/changed?

The current quest setup was slightly confusing, so if you don't look at the Theurgy guide book, you don't realize you need the stuff from some dependent quests before you can make the incubator. Also, it doesn't really make sense to make a fermentation vat before making a mercury distiller.

**Comment by Morelyn** [comment]
Indeed... also you want the reformation array after the mercury catalyst and the sulfuric flux emitter that are necessary to power the array
